src/jobs/index.html

```diff
@@ -61,201 +61,55 @@
 
 
 <article class="container-md">
-
-  <h1>Join Our Team</h1>
-
-  <p>MyCrypto is one of the world’s leading interfaces for the Ethereum blockchain, offering a suite of tools helpful for any user - from someone who is completely new to the space to the most tenured of crypto enthusiasts. We’re looking to advance our offering to other blockchains and other platforms, all while developing new solutions to meet people’s needs.</p>
-  <h5>Our Products</h5>
-  <ul>
+    <h1>Join Our Team</h1>
+
+    <p>MyCrypto is one of the world’s leading interfaces for the Ethereum blockchain, offering a suite of products for all types of people. We strive to build things that <em>actually</em> solve real problems and iterate quickly. We take an holistic approach to improving this ecosystem and people's lives, prioritizing security, customer support, and being part of this amazing community along with building amazing products.</p>
```

Comment:
"We strive to build things that <em>actually</em> solve real problems and iterate quickly."

change to

"We strive to build things that <em>actually</em> solve real problems, and we iterate quickly."